src/main/java/uk/ac/ic/wlgitbridge/bridge/repo/FSGitRepoStore.java

```diff
@@ -22,10 +23,16 @@
 
     private final String repoStorePath;
     private final File rootDirectory;
+    private final Function<File, Long> fsSizer;
 
     public FSGitRepoStore(String repoStorePath) {
+        this(repoStorePath, d -> d.getTotalSpace() - d.getFreeSpace());
```

Comment:
Will this call work, given that the partition we're interested in is:

```
/dev/xvdf      515930552 257808644 231891124  53% /var/git-bridge
```

and git-bridge config says

```
"rootGitDirectory": "/var/git-bridge/git",
```

or do we need to translate `/var/git-bridge/git` to the mount point of its partition `/var/git-bridge` and from there to the partition name `/dev/xvdf` when making this call?